src/config.test.js

```diff
@@ -10,40 +10,40 @@ describe('Config', () => {
 
   beforeEach(async () => {
     resetConfig()
-    delete process.env[`TENANT_SEED_${tenantName}`]
-    delete process.env[`TENANT_DIDMETHOD_${tenantName}`]
```

Comment:
Instead of manipulating the process.env, is it possible to create a getter function that gets data for a tenant from process.env and then make a mock implementation in tests?